<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSelectVerse</class>
 <widget class="QDialog" name="DialogSelectVerse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Book, Chapter, Verse</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>166</width>
     <height>76</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Book</string>
   </property>
   <widget class="QComboBox" name="comboBook">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>146</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>96</width>
     <height>76</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Chapter</string>
   </property>
   <widget class="QComboBox" name="comboChapter">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>30</y>
      <width>66</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>96</width>
     <height>76</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Verse</string>
   </property>
   <widget class="QComboBox" name="comboVerse">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>30</y>
      <width>66</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>25</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
